### PR TITLE
Sync state to sessionStorage on query change

### PR DESCRIPTION
**Pull request by eckdanny-osi**

### CODE REVIEW

File: app/javascript/store/configureStore.js [hunk 1]
```diff
@@ -2,18 +2,45 @@ import createSagaMiddleware from 'redux-saga';
...
+import { saveState, loadState } from '../_utils/sessionStore';
+import { getSearchParams } from '../selectors/userListSelector';
 import reducer from '../reducers';
 
+export const store = configureStore();
+
+if (process.env.ENABLE_SESSION_STORAGE_SYNC)
+  observeStore(store, getSearchParams, () => saveState(store.getState()));
+
+//
```
Comment (kelfink) [inline app/javascript/store/configureStore.js]: lots of white space?

File: app/javascript/store/configureStore.js [hunk 2]
```diff
@@ -2,18 +2,45 @@ import createSagaMiddleware from 'redux-saga';
...
+//
+//
+
 function configureStore(initialState) {
   const sagaMiddleware = createSagaMiddleware();
+  const state = process.env.ENABLE_SESSION_STORAGE_SYNC
+    ? { ...initialState, ...loadState() }
```
Comment (kelfink) [inline app/javascript/store/configureStore.js]: nice.  that looks tidy.

File: app/javascript/views/usersList/UsersList.jsx [hunk 3]
```diff
@@ -180,7 +180,7 @@ class UserList extends PureComponent {
                 headerBtnName="+ Add a user"
                 onEdit={this.handleOnAdd}
               >
-                <form onSubmit={this.submitSearch}>
+                <form onSubmit={this.submitSearch} autoComplete="off">
```
Comment (kelfink) [inline app/javascript/views/usersList/UsersList.jsx]: nice touch.  Was it causing a problem?

Comment (eckdanny-osi) [reply app/javascript/views/usersList/UsersList.jsx]: must've dropped a commit or something in the original topic branch. I never meant for autoComplete to be enabled.

**Pull request merged**